src/praisonai/praisonai/tools/skill_manage.py

```
@@ -0,0 +1,428 @@
+            return self.create(name + "_edit", content, propose=True)
+        else:
+            skill_path = self._find_skill(name)
+            if not skill_path:
+                return f"❌ Skill '{name}' not found."
+            
+            try:
+                skill_md = skill_path / "SKILL.md"
+                skill_md.write_text(content)
+                self._log_action("edit", name, False)
+                return f"✅ Skill '{name}' content updated."
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)